frontend/components/viewer/js/viewer.service.ts

```diff
@@ -186,15 +186,13 @@ export class ViewerService {
 		});
 	}
 
-	public getCurrentViewpoint(params) {
+	public async getCurrentViewpoint({ account, model, promise }) {
```

Comment:
I'm guessing this is only taking a promise because issues isn't changed?

When it's all changed over can we also make it to take 2 parameters instead of an object with 2 parameters?

Reply:
Yes and yes ;)